assets/helpers/contributions.js

```diff
@@ -0,0 +1,22 @@
+// @flow
```

Comment:
If this file only contains type definition I would call it `contributionsType` or even I would put it inside a Types folder. I guess that the helper folder should be for functionality.